lib/cartopy/crs.py

```diff
@@ -1212,8 +1218,8 @@ def ellipse(semimajor=2, semiminor=1, easting=0, northing=0, n=200):
             point = sgeom.Point(false_easting, false_northing)
             self._boundary = point.buffer(self._x_limits[1]).exterior
         else:
-            coords = ellipse(self._x_limits[1], self._y_limits[1],
-                             false_easting, false_northing, 90)
+            coords = _ellipse_boundary(self._x_limits[1], self._y_limits[1],
+                                       false_easting, false_northing, 91)
```

Comment:
`np.linspace(0, 2*pi, even)` does not touch pi/2, pi or 3pi/2 so it doesn't really reach the x/y limits.